documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -14,290 +14,210 @@ keywords:
-- `JcsEd25519Signature2020` 
+| Name      | Description                                     |
+|-----------|-------------------------------------------------|
+| ByteArray | A dynamically sized, but unprefixed byte array. |
 
-It must be emphasized that the IOTA Identity implementation of  `JcsEd25519Signature2020` follows the [official specification](https://identity.foundation/JcsEd25519Signature2020/) with the single exception that the SHA-256 pre-hash of the input, the third step of the [proof generation algorithm](https://identity.foundation/JcsEd25519Signature2020/#ProofGeneration), is skipped. See the following two GitHub issues [#22](https://github.com/decentralized-identity/JcsEd25519Signature2020/issues/22) and [#26](https://github.com/decentralized-identity/JcsEd25519Signature2020/issues/26) for more context on why this deviation is deemed necessary. 
 
-**Verification Method**
+#### Payload
 
-The proof object MUST include a `verificationMethod` which references a verification method embedded in the same DID Document. The verification method must be capable of signing DID messages as per the [Signing Keys](#signing-keys) section.
+The payload must contain the following fields:
 
-**Signature**
+* `metadata`: contains metadata about the DID document. For example, `created` to indicate the time of
+  creation, and `updated` to indicate the time of the last update to the document. It can also include other properties.
+* `document`: which contains the DID document. In the example below, the document only contains one verification method. The `id` is specified by `did:0:0` which references the DID of the document itself, since the `id` is unknown at the time of publishing. It also duplicates the DID of the document to reduce the size of the state metadata, in turn reducing the required storage deposit.
 
-Depending on the verification method, a set of fields are REQUIRED to provide the cryptographic proof of the DID Document. For example, the `JcsEd25519Signature2020` method has a `signatureValue` field. 
+Example State Metadata Document:
 
-Example `proof` using the `JcsEd25519Signature2020` method:
-```json
-"proof": {
-    "type": "JcsEd25519Signature2020",
-    "verificationMethod": "did:iota:GzXeqBXGCbuebiFtKo4JDNo6CmYmGbqxyh2fDVKadiBG#authentication",
-    "signatureValue": "3fLtv3KUU4T5bHNLprV3UQ2Te3bcRZ9uUYSFouEA7fmYthieV35NNLqbKUu8t2QmzYgnfp1KMzCqPzGNi3RjU822"
+```Json
+{
+   "document":{
+      "id":"did:0:0",
+      "verificationMethod":[
+         {
+            "id":"did:0:0#key-1",
+            "controller":"did:0:0",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z6BNtMbKY78XDVuqfh4u15bZkByu94XNVr9RpqEGCNncn"
+         }
+      ]
+   },
+   "metadata":{
+      "created":"2022-08-02T21:39:48Z",
+      "updated":"2022-08-02T21:39:48Z"
+   }
 }
 ```
 
-### Anatomy of the Diff object
+In a state metadata document, every occurrence of the DID must be replaced by `did:0:0`.
```

Comment:
Actually, let's state this in the table, as suggested in another comment, and remove this so it's all in one place.

```suggestion
```